website/docs/components/link/inline/partials/code/how-to-use.md

```diff
@@ -3,37 +3,47 @@
 The most basic invocation requires some content to be passed as children and either an `@href` or `@route` argument.
 
 ```handlebars
-<Hds::Link::Inline @href="...">Watch tutorial video</Hds::Link::Inline>
+<div class="hds-typography-body-300">
+  Lorem <Hds::Link::Inline @href="...">ipsum dolor</Hds::Link::Inline> sit amet consectetur adipiscing elit.
+</div>
 ```
 
 ### Color
 
-There are two available colors for a Inline Link: `primary` and `secondary`. The default is `primary`. To use a different color, set `@color` to `secondary`.
+There are two available colors for an Inline Link: `primary` and `secondary`. The default is `primary`. To use a different color, set `@color` to `secondary`.
 
 ```handlebars
-<Hds::Link::Inline @color="primary" @href="...">Read tutorial</Hds::Link::Inline>
+<div class="hds-typography-body-300">
```

Comment:
Probably to avoid users thinking that they should use a `<div>` for text (in general) I would use a `<span>` instead (the visual presentation results to be the same).

What do you think?

Reply:
Yep, that works for me. I thought I had tried that initially but you're right it does present the same.

Reply:
Sorry I didn't see this in time. To me a `span` doesn't make any more sense than a `div`. I most likely would have used `p` tags. Anyway I guess it doesn't make a huge difference...